org-code-javabuilder/media/src/main/java/org/code/media/SoundLoader.java

```diff
@@ -10,6 +17,43 @@
    * @throws SoundException when the file cannot be found
    */
   public static double[] read(String filename) throws SoundException {
```

Comment:
I wonder if we should throw a `FileNotFoundException` if the file isn't found and a `SoundException` if there's an issue with the file? This would have consistency with how we plan to handle exceptions for images.

Reply:
Yeah, makes sense, I'll add that